README.md

```diff
@@ -1072,6 +1073,176 @@ Bear in mind that with the introduction of the new V8 engine alongside the new E
 
 <br/><br/><br/>
 
+<p align="right"><a href="#table-of-contents">⬆ Return to top</a></p>
+
+# `7. Draft: Docker Best Practices`
+
+## Our contributors are working on this section. [Would you like to join?](https://github.com/goldbergyoni/nodebestpractices/issues/682)
+
+<br/><br/>
+
+## ![✔] 8.1. Clean npm cache
+
+**TL;DR:**
+
+**Otherwise:**
+
+🔗 [**Read More: Clean npm cache**](/sections/docker/file.md)
+
+<br /><br /><br />
+
+## ![✔] 8.2. Bootstrap the code using 'node' command, avoid 'npm run' scripts
+
+**TL;DR:**
+
+**Otherwise:**
+
+🔗 [**Read More: Clean npm cache**](/sections/docker/file.md)
+
+<br /><br /><br />
+
+## ![✔] 8.3. Install packages for production
+
+**TL;DR:**
+
+**Otherwise:**
+
+🔗 [**Read More: Bootstrap the code using 'node' command, avoid 'npm run' scripts**](/sections/docker/file.md)
+
+<br /><br /><br />
+
+## ![✔] 8.4. Lint your Dockerfile
+
+**TL;DR:**
+
+**Otherwise:**
+
+🔗 [**Read More: Lint your Dockerfile**](/sections/docker/file.md)
+
+<br /><br /><br />
+
+## ![✔] 8.5. Utilize caching for better build time
+
+**TL;DR:**
+
+**Otherwise:**
+
+🔗 [**Read More: Utilize caching for better build time**](/sections/docker/file.md)
+
+<br /><br /><br />
+
+## ![✔] 8.6. Set Docker memory limits which are in-par with v8 memory limit
+
+**TL;DR:**
+
+**Otherwise:**
+
+🔗 [**Read More: Set Docker memory limits which are in-par with v8 memory limit**](/sections/docker/file.md)
+
+<br /><br /><br />
+
+## ![✔] 8.7. Scan your image for vulnerabilities
+
+**TL;DR:**
+
+**Otherwise:**
+
+🔗 [**Read More: Scan your image for vulnerabilities**](/sections/docker/file.md)
+
+<br /><br /><br />
+
+## ![✔] 8.8. Use multistage builds
+
+**TL;DR:**
+
+**Otherwise:**
+
+🔗 [**Read More: Use multistage builds**](/sections/docker/file.md)
+
+<br /><br /><br />
+
+## ![✔] 8.9. Don't use "latest" tags, use a digest
+
+**TL;DR:**
+
+**Otherwise:**
+
+🔗 [**Read More: Don't use "latest", use a digest**](/sections/docker/file.md)
+
+<br /><br /><br />
+
+## ![✔] 8.10. Prefer smaller images
+
+**TL;DR:**
+
+**Otherwise:**
+
+🔗 [**Read More: Prefer smaller images**](/sections/docker/file.md)
+
+<br /><br /><br />
+
+## ![✔] 8.11. Graceful shutdown
+
+**TL;DR:**
+
+**Otherwise:**
+
+🔗 [**Read More: Graceful shutdown**](/sections/docker/file.md)
+
+<br /><br /><br />
+
+## ![✔] 8.12. Avoid sending secrets as build time arguments
+
+**TL;DR:**
+
+**Otherwise:**
+
+🔗 [**Read More: Avoid sending secrets as build time arguments**](/sections/docker/file.md)
+
+<br /><br /><br />
+
+## ![✔] Use .dockerignore to prevent leaking secrets
```

Comment:
Could we consider changing the bullet title to  Use .dockerignore files? Or similar?

I think that the point of the dockerignore file determining what is sent to the build daemon is more than just a side bonus - it is equally as important as the leaking secrets point.

So making the title more generic might be a nice enhancement

Reply:
@js-kyle Isn't a generic title goes against your main point: By focusing on the security aspect of it, I'm emphasizing how critical it is. In overall, I think we should always emphasize the 'WHY'. Don't you feel that writing 'Use .dockerignore files' lack the reason of doing so?